recipe/cp_libtensorflow_cc.sh

```diff
@@ -1 +1,3 @@
 tar -C ${PREFIX} -xf $SRC_DIR/libtensorflow_cc_output.tar
+mkdir -p ${PREFIX}/include/tensorflow/tsl/platform/include
+cp float8.h ${PREFIX}/include/tensorflow/tsl/platform/include
```

Comment:
Its somewhat strange because ml_dtypes is a python package. we could repackage ml_dtypes as a library.

Reply:
This is in general a very strange interaction as Tensorflow also builds "a" ml_dtypes as a library internally again. I would rather move forward with this hack and hope that in 2.15 things are cleared up a bit more instead of trying to split the `ml_dtypes` package into pieces.

Reply:
do they vendor the library? it seems scary to pin so hard on ml_dtypes 0.2.0